protocol/utils.go

```diff
@@ -45,24 +45,31 @@ func AssetSymbol(b backend.Backend, asset types.EthAsset) (string, error) {
 }
 
 // CheckSwapID checks if the given log is for the given swap ID.
-func CheckSwapID(log *ethtypes.Log, eventName string, contractSwapID types.Hash) error {
-	abiSF, err := abi.JSON(strings.NewReader(contracts.SwapFactoryMetaData.ABI))
-	if err != nil {
-		return err
-	}
+func CheckSwapID(log *ethtypes.Log, eventNameTopic [32]byte, contractSwapID types.Hash) error {
+	// abiSF, err := abi.JSON(strings.NewReader(contracts.SwapFactoryMetaData.ABI))
+	// if err != nil {
+	// 	return err
+	// }
 
-	data := log.Data
-	res, err := abiSF.Unpack(eventName, data)
-	if err != nil {
-		return err
-	}
+	// data := log.Data
+	// res, err := abiSF.Unpack(eventName, data)
+	// if err != nil {
+	// 	return err
+	// }
+
+	// if len(res) < 1 {
+	// 	return errLogMissingParams
+	// }
```

Comment:
Can this commented out code be deleted? Or do we need it for some near term additional change?